### PR TITLE
feat: added warning if app already deployed

### PR DESCRIPTION
**Pull request by raaymax**

### CODE REVIEW

File: src/writer/deploy.py [hunk 1]
```diff
@@ -15,6 +15,7 @@
 WRITER_DEPLOY_URL = os.getenv("WRITER_DEPLOY_URL", "https://api.writer.com/v1/deployment/apps")
 
 def deploy(path, token, env):
+    check_app(token)
```
Comment (FabienArcellier) [inline src/writer/deploy.py:18]: You should add a quiet or force flag for continuous delivery

**Comment by ramedina86** [comment]
I think the point raised by @FabienArcellier is important.

@raaymax I'm guessing it makes more sense to merge this PR and then implement in CLI refactor PR though, right?

**Comment by raaymax** [comment]
agree, with both 
- `--quitet` is important I will add it.
- first merge this and then add it to click

**Comment by raaymax** [comment]
@ramedina86 can we merge this one?

**Comment by ramedina86** [comment]
I was just waiting for `--quiet` has this been implemented?

**Comment by raaymax** [comment]
Ach a little miss understanding 😅 I will add it to click PR

**Pull request merged**